package.json

```diff
@@ -60,11 +61,10 @@
     "format-js": "eslint lib test --fix",
     "check:lint": "eslint lib test",
     "precommit": "check-clang-format",
-    "test": "mocha test",
     "docs": "jsdoc2md --template etc/README.hbs --plugin dmd-clear --files lib/kerberos.js > README.md",
-    "rebuild": "prebuild --runtime napi --compile",
-    "prebuild": "prebuild --runtime napi --strip --verbose --all",
-    "release": "standard-version -i HISTORY.md"
+    "test": "mocha test",
+    "prepare": "tsc || true",
```

Comment:
I can take this TS stuff out, I am aiming to align the native packages here, and this is a starting point.